README.md

```diff
@@ -1 +1,49 @@
 # OTP Builds
+
+This is a collection of community-maintained [Erlang/OTP](https://github.com/erlang/otp) binary builds.
+
+Supported operating systems:
+
+  * `macos`
+
+Supported architectures:
+
+  * `amd64`
+  * `arm64`
+
+## List of Builds
+
+List of builds can be found here:
```

Comment:
Should this mention that `master` and `maint*` are also available but they "move" over time?

Reply:
Yup, done!